ci/properties/tencent.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Build and Deploy to TKE",
+    "description": "Build a Docker container, publish it to TKE Registry, and deploy to TKE cluster.",
+    "iconName": "tencentcloud",
```

Comment:
Can you add an SVG icon to the `./icons` folder with this name please?

Reply:
done